templates/drupal10/ce-dev/ansible/deploy.yml.j2

```diff
@@ -0,0 +1,74 @@
+---
+# Template playbook for a local Drupal 9 codebase.
```

Comment:
A sneaky *Drupal 9* that needs updating.

Reply:
Updated PR